mockserver-core/src/test/java/org/mockserver/server/initialize/ExpectationInitializerLoaderTest.java

```diff
@@ -216,14 +216,14 @@ public void shouldLoadExpectationsFromFileSystemInJson() throws Exception {
                 )
             };
             File mockserverInitializer = File.createTempFile("mockserverInitialization", ".json");
-            Files.write(mockserverInitializer.toPath(), expectationSerializer.serialize(expections).getBytes(StandardCharsets.UTF_8));
+            Files.write(mockserverInitializer.toPath(), expectationSerializer.serialize(expectations).getBytes(StandardCharsets.UTF_8));
             ConfigurationProperties.initializationJsonPath(mockserverInitializer.getAbsolutePath());
 
             // when
-            final Expectation[] expectations = new ExpectationInitializerLoader(configuration(), new MockServerLogger(), mock(RequestMatchers.class)).loadExpectations();
+            final Expectation[] loadedExpectations = new ExpectationInitializerLoader(configuration(), new MockServerLogger(), mock(RequestMatchers.class)).loadExpectations();
 
             // then
-            assertThat(expectations, is(expections));
+            assertThat(expectations, is(loadedExpectations));
```

Comment:
This section has two `expectations`, the one from the top of the function, and then a second one. Since it's comparing the pair, I've renamed the second one to `loadedExpectations` -- I'm definitely not attached to the name...